fwup.conf

```diff
@@ -136,6 +136,12 @@ file-resource rootfs.img {
     # Error out if the rootfs size exceeds the partition size
     assert-size-lte = ${ROOTFS_A_PART_COUNT}
 }
+file-resource i2c-bcm2708.dtbo {
```

Comment:
I'm looking at https://github.com/raspberrypi/linux/blob/rpi-5.4.y/arch/arm/boot/dts/overlays/i2c-bcm2708-overlay.dts, and I'm not sure why this is needed. Does the Speaker Bonnet work without it?